plugins/inputs/prometheus/README.md

```diff
@@ -11,8 +11,14 @@ in Prometheus format.
   ## An array of urls to scrape metrics from.
   urls = ["http://localhost:9100/metrics"]
 
-  ## Metric version (optional, default=1, supported values are 1 and 2)
-  # metric_version = 2
+  ## Metric version controls the mapping from Prometheus metrics into
+  ## Telegraf metrics.  When using the prometheus_client output, use the same
+  ## value in both plugins to ensure metrics are round-tripped without
+  ## modification.
+  ##
+  ##   example: metric_version = 1; deprecated in 1.13
+  ##            metric_version = 2; recommended version
+  # metric_version = 1
```

Comment:
If 1 is deprecated, shouldn't we default to 2?

Reply:
We haven't been doing this because it would switch over users automatically when they update Telegraf.  For most users this would break their dashboards/alerts.

I was planning that when Telegraf 2.0 is released, we would switch the default and remove the old version in one go.  I think what this means is that we ought to be logging something here on startup when the user is still on the deprecated version.

Reply:
I added a log message into the init function of both the prometheus input and output which looks like:
```
W! [inputs.prometheus] Use of deprecated configuration: 'metric_version = 1'; please update to 'metric_version = 2'
```

Reply:
I also updated the deprecation docs: https://github.com/influxdata/telegraf/wiki/Deprecation#deprecate-options